pystiche_papers/ulyanov_et_al_2016/_loss.py

```diff
@@ -86,6 +96,12 @@ def calculate_score(
             return score
 
         batch_size = input_repr.size()[0]
+        # instance_norm:
```

Comment:
1. I would move this into `__init__` and above `self.double_batch_size_mean`
2. You don't need to make distinction between `instance_norm` and `not instance_norm` when both have the same behavior. Just include both links.